src/sentry/static/sentry/app/components/events/sdkUpdates.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SentryTypes from 'app/sentryTypes';
+import Alert from 'app/components/alert';
+import ExternalLink from 'app/components/links/externalLink';
+
+import {t} from 'app/locale';
+
+export const EnableIntegrationSuggestion = PropTypes.shape({
+  type: PropTypes.string,
+  integrationName: PropTypes.string,
+  integrationUrl: PropTypes.string,
+});
+
+export const UpdateSdkSuggestion = PropTypes.shape({
+  type: PropTypes.string,
+  sdkName: PropTypes.string,
+  newSdkVersion: PropTypes.string,
+  sdkUrl: PropTypes.string,
+});
+
+export const ChangeSdkSuggestion = PropTypes.shape({
+  type: PropTypes.string,
+  newSdkName: PropTypes.string,
+  sdkUrl: PropTypes.string,
+});
+
+export const Suggestion = PropTypes.oneOfType([
+  EnableIntegrationSuggestion,
+  UpdateSdkSuggestion,
+  ChangeSdkSuggestion,
+]);
+
+function getSuggestionComponentKey(suggestion) {
+  return JSON.stringify(suggestion, Object.keys(suggestion).sort());
+}
+
+class EventSdkUpdateSuggestion extends React.Component {
+  static propTypes = {
+    event: SentryTypes.Event.isRequired,
+    suggestion: Suggestion.isRequired,
+  };
+
+  getSuggestionTitle() {
+    const {event, suggestion} = this.props;
+    let href;
+    let content;
+    switch (suggestion.type) {
+      case 'updateSdk':
+        href = suggestion.sdkUrl;
+        content = t(
+          'Update your SDK from version %s to version %s',
+          event.sdk.version,
+          suggestion.newSdkVersion
+        );
+        break;
+      case 'changeSdk':
+        href = suggestion.sdkUrl;
+        content = t("Migrate to the '%s' SDK", suggestion.newSdkName);
```

Comment:
Might look cleaner with a different styling to the sdk/integration name instead of a single quote, e.g.

```suggestion
        content = tct("Migrate to the [sdkName] SDK", {
          sdkName: <code>{suggestion.newSdkName}</code>
        });
```